packages/flutter/lib/src/services/text_input.dart

```diff
@@ -13,6 +14,7 @@ import 'dart:ui' show
   TextDirection;
 
 import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart' show View;
```

Comment:
This looks like a layering violation. The `services` layer is below `widgets` and therefore it cannot import `widgets`.

Reply:
Looks like you're not actually using `View` outside of docs. So, everything should still work if you just remove this import.

Reply:
That's right, I'm only importing it for the docs.

Without the import, VS Code can't navigate to the definition. How do you usually deal with this?

Reply:
> Without the import, VS Code can't navigate to the definition.

Yeah, that is a current limitation of dartdoc. Afaik, there is no work around. You could upvote https://github.com/dart-lang/sdk/issues/49073 and/or https://github.com/dart-lang/sdk/issues/50702 to get this fixed in dartdocs.